playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx

```diff
@@ -31,6 +32,7 @@ type MultiLevelSelectProps = {
   treeData?: { [key: string]: string }[]
   onSelect?: (prop: { [key: string]: any }) => void
   selectedIds?: string[]
+  // ultimateChildrenOnly?: boolean
```

Comment:
Looks great @Israel-Molestina! Don't forget to remove the commented out code if you don't need it anymore!